cmd/cmp/main.go

```diff
@@ -52,23 +52,34 @@ func Run(_ *cobra.Command, args []string) {
 		}
 		if match := fileIdRegex.FindStringSubmatch(entry.Name()); len(match) > 0 {
 			fileId := match[1]
-			probeSideFileCount++
 			buildSideFilePath, exists := fileIdMap[fileId]
 			if !exists {
 				continue
 			}
+			probeSideFileCount++
```

Comment:
This may be renamed to something like `probeSideMatched`. I think that will make more sense..